examples/xdamage.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+#
+# examples/xdamage.py -- demonstrate damage extension
+#
+#    Copyright (C) 2002 Peter Liljenberg <petli@ctrl-c.liu.se>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation; either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+#    Free Software Foundation, Inc.,
+#    59 Temple Place,
+#    Suite 330,
+#    Boston, MA 02111-1307 USA
+
+
+# Python 2/3 compatibility.
+from __future__ import print_function
+
+import sys
+import os
+
+# Change path so we find Xlib
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+from Xlib import display, X, threaded,Xutil
+import time
+import thread
+from Xlib.ext import damage
+from PIL import Image, ImageTk
+
+def redraw(win, gc):
+    # win.clear_area()
+    win.fill_rectangle(gc, 0, 0, 60, 60)
+
+def blink(display, win, gc, cols):
+    while 1:
+        time.sleep(2)
+        print('Changing color', cols[0])
+        gc.change(foreground = cols[0])
+        cols = (cols[1], cols[0])
+        redraw(win, gc)
+        display.flush()
+
+def getImageFromWin(win,ptW,ptH,ptX=0,ptY=0):
```

Comment:
`python-xlib` generally follows PEP8 style except names from protocol. So it's better to adjust this and below functions to such a signature:
```python
def get_image_from_win(win, pt_w, pt_h, pt_x=0, pt_y=0):
```